pkgs/development/tools/go-langserver/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "go-langserver-${version}";
+  version = "unstable-2018-03-05";
+  rev = "5d7a5dd74738978d635f709669241f164c120ebd";
+
+  goPackagePath = "github.com/sourcegraph/go-langserver";
+  subPackages = [ "." ];
+
+  src = fetchgit {
```

Comment:
Could you use fetchFromGithub (in order to fetch a tgz instead of a git repository)?

Reply:
I wasn't aware of this behavior, interesting! I have changed it to fetchFromGitHub as requested.

Reply:
Have made the requested changes and added meta information!